gradio/components.py

```diff
@@ -2519,8 +2519,7 @@ def postprocess(self, y: str | pd.DataFrame) -> Dict:
         if y is None:
             return None
         if isinstance(y, str):
-            y = pd.read_csv(y)
-            return {"headers": y.columns.values.tolist(), "data": y.values.tolist()}
```

Comment:
All return values have to have headers now.

Reply:
Oh maybe this is only true for Dataframe, ideally we'd make the interfaces consistent since they are very similar. What is the reason for this change, how does it address the issue?

Reply:
Removed this change, and instead changed the frontend to convert the timeseries obj to string (to fix examples).